### PR TITLE
show files uploaded and info submitted in new app. acknowledgement email

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story: Email to acknowledge new app: remind if no files uploaded
https://www.pivotaltracker.com/story/show/152441894

### Changes proposed in this pull request:
1.  display info from an application in the email sent out to the applicant
2. added method to get File Delivery description for a particular locale since it's now required in 2 places (both the shf application view _and_ now this email) `AdminOnly::FileDeliveryMethod description_for_locale(locale)`

### Screenshots (Optional):
**sv: no emails uploaded:**

email-new-app-ack-no-uploads-sv1
**sv: some emails uploaded:**

email-new-app-ack-with-uploads-sv1

---

**en: no emails uploaded:**
email-new-app-ack-no-uploads-en1

**en: some emails uploaded:**
email-new-app-ack-with-uploads-en1

### Ready for review:
@thesuss  -- wording.  (also: I put CSS classes in the emails so hopefully they are easy to format.  If you have suggestions, let me know those, too.  That's easy to change at a later date, of course.  No hurry on my part for that now.)
@patmbolger

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -529,12 +529,13 @@ sv:
...
-      filename: Filnamn
-      file_was_uploaded: 'Filen laddades upp: %{filename}'
-      confirm_delete: 'Är du säker på att du vill radera %{filename}?'
+      filename: &filename Filnamn
+      file_was_uploaded: &file_was_uploaded 'Filen laddades upp: %{filename}'
+      confirm_delete: &upload_confirm_delete 'Är du säker på att du vill radera %{filename}?'
       invalid_upload_type: &invalid_upload_type Tyvärr kan du inte ladda upp denna filtypen.
       file_too_large: &file_too_large_5MB Filen måste vara mindre än 5 MB
       please_upload_again: Var god ladda upp dina filer igen.
+      will_review_after_received: &review_after_recevied_files "Once we have received your files, we will review your application."
```
Comment (thesuss) [inline config/locales/sv.yml]: Vi kommer att handlägga din ansökan när vi har fått alla filer.

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1037,6 +1037,9 @@ sv:
       acknowledge_received:
         subject: Din ansökan till Sveriges Hundföretagare är mottagen
         message_text: "Din ansökan är mottagen. Vi hanterar den så fort vi kan. Säkerställ att du laddat upp eller mailar in allt relevant underlag."
+        your_submitted_info: "As a reminder, here is what you submitted:"
```
Comment (thesuss) [inline config/locales/sv.yml]: Här är vad du har laddat upp:

File: config/locales/sv.yml [hunk 3]
```diff
@@ -1037,6 +1037,9 @@ sv:
...
+        review_after_recevied_files: *review_after_recevied_files
+        upload_choice_intro: "Your choice for providing files: "
```
Comment (thesuss) [inline config/locales/sv.yml]: Ditt val för att tillhandahålla underlag:

File: app/views/shf_application_mailer/acknowledge_received.html.haml [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+  .shf-application-info
+    %p
+      = shf_app_to_html(@shf_application)
+  %hr
+
+  - if @shf_application.uploaded_files.size == 0
+    %p.nofiles-will-review= t('review_after_recevied_files', scope: i18n_scope)
+
+  %p.login-url
+    = 'You can log in and change your information and upload more files. '
```
Comment (thesuss) [inline app/views/shf_application_mailer/acknowledge_received.html.haml]: oh my got its not a t()

Comment (weedySeaDragon) [reply app/views/shf_application_mailer/acknowledge_received.html.haml]: oh the horror!!!

File: app/models/admin_only/file_delivery_method.rb [hunk 5]
```diff
@@ -28,7 +28,7 @@ def self.get_method(name_key)
         raise ArgumentError, 'Argument must be a symbol and a known delivery name key'
       end
 
-      where(name: METHOD_NAMES[name_key])[0]
+      where(name: METHOD_NAMES[name_key])[0]  # TODO: should use :find instead of :where
```
Comment (patmbolger) [inline app/models/admin_only/file_delivery_method.rb]: You're right - why not make that small change in this PR?

**Review by patmbolger — COMMENTED** [review]
Looks good - just one comment.

**Review by patmbolger — APPROVED** [review]

File: app/views/shf_application_mailer/acknowledge_received.html.haml (hunk 4 above)
Comment (weedySeaDragon) [reply app/views/shf_application_mailer/acknowledge_received.html.haml]: @thesuss  - I've updated the locale info and even made this into a t() ;-)  
Ready for you to review it again to make sure I didn't blorf something up.

(then if all is ok, squash and merge it!)

**Pull request merged**